src/App.js

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import './App.css';
 import { Home, About, Contact, Navigation } from './components';
+import { Route } from 'react-router-dom';
 
 const App = () => (
   <div>
     <Navigation />
+    <Route exact path="/" component={Home} />
+    <Route path="/about" component={About} />
+    <Route path="/Contact" component={Contact} />
```

Comment:
be mindful of keeping capitalization consistent. Otherwise you might confuse yourself later